src/modules/commander/Commander.cpp

```diff
@@ -2729,12 +2729,15 @@ Commander::set_main_state_rc(const vehicle_status_s &status_local, bool *changed
 		|| (_last_manual_control_setpoint.stab_switch != _manual_control_setpoint.stab_switch)
 		|| (_last_manual_control_setpoint.man_switch != _manual_control_setpoint.man_switch);
 
-	// only switch mode based on RC switch if necessary to also allow mode switching via MAVLink
-	const bool should_evaluate_rc_mode_switch = first_time_rc
```

Comment:
Did you remove the `first_time_rc` because it anyways triggers the `some_switch_changed` / new `should_evaluate_rc_mode_switch`?

Reply:
Yes I couldn't think of a case where it would matter because all the switches in `_last_manual_control_setpoint` are initially `SWITCH_POS_NONE`. Did I miss anything?